### PR TITLE
Add case study edit and delete

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: app/helpers/react/response.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def from_response(request):
```
Comment (matt-sm) [inline app/helpers/react/response.py]: need to copy this to utils

File: app/main/views/suppliers.py [hunk 2]
```diff
@@ -93,21 +94,34 @@ def new_supplier_case_study():
...
 
-    casestudy = {}
-    for key in request.form.keys():
-        if key not in ['csrf_token']:
-            value = request.form.getlist(key)
-            if len(value) == 1 and key not in ['projectLinks', 'outcome']:
-                casestudy[key] = value[0]
-            else:
-                casestudy[key] = value
+    casestudy = from_response(request)
```
Comment (matt-sm) [inline app/main/views/suppliers.py]: much better

File: tests/app/views/test_case_studies.py [hunk 3]
```diff
@@ -119,3 +119,49 @@ def test_create_new_work_order(self, data_api_client):
...
+        ).start()
+
+        self._render_component = mock.patch(
+            'app.main.suppliers.render_component'
+        ).start()
+
+        self.case_study = self._get_case_study_fixture_data()
+
+    def teardown(self):
+        self._data_api_client.stop()
```
Comment (matt-sm) [inline tests/app/views/test_case_studies.py]: was curious why you call stop() on teardown?

Comment (maxious) [inline tests/app/views/test_case_studies.py]: Dunno, was just copying https://github.com/AusDTO/dto-digitalmarketplace-buyer-frontend/blob/master/tests/app/views/test_marketplace.py#L113 etc.

**Pull request merged**